### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by VicPeralta**
# Bookstore: initialize the project with components

In this first milestone:

- Used React Router.
- Used React components.
- No linter errors.
- Correct GitFlow.
- Repo documented in a professional way.
- Initialized a CRA app.
- A directory `components` was created to host all components.
- Two routes were created (Books and Categories)
- Books is the default view, it contains a list of books (just one book) with buttons for Comments, Edit, and Remove with no functionality.
- Books view contains a form for adding a book, no functionality provided yet.
- Categories view just displays a button.
- No major styling was used.

### CODE REVIEW

File: src/components/Book/Book.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import './Book.css';
+
+const Book = () => (
+  <div className="book">
+    <p>Action</p>
+    <h3>The Hunger Games</h3>
+    <p>Suzanne Collins</p>
```
Comment (FrederickMih) [inline src/components/Book/Book.js]: - [ ] Nice job for displaying the title and author of the book however requirements say the `Book` should take the book data as a prop and renders the title and the author of the book. Please let's considered this requirement too.

File: src/components/Input-Form/Input-Form.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  <form className="input-form">
+    <label htmlFor="title">
+      Title
+      <input type="text" name="title" id="title" required />
+    </label>
+    <label htmlFor="author">
+      Author
+      <input type="text" name="author" id="author" required />
+    </label>
+    <input type="submit" />
```
Comment (FrederickMih) [inline src/components/Input-Form/Input-Form.js]: - [ ] Please can we make this a button?. The requirements of this project stipulate that this should be a button.

File: src/components/NavBar/NavBar.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    id: 1,
+    path: '/',
+    text: 'BOOKS',
+  },
+  {
+    id: 2,
+    path: '/categories',
+    text: 'CATEGORIES',
+  },
+];
```
Comment (FrederickMih) [inline src/components/NavBar/NavBar.js:16]: - [ ] I found this piece of code interesting. Keep up!

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @VicPeralta , 

You have done a fantastic job on this project so far!
You are left with little issues that you still need to work on to go to the next project but you are almost there!


# To Highlights :rocket: 

- Followed GitHub flow :heavy_check_mark: 
- All linters checks are passing :heavy_check_mark: 
- Your README file is professional and descriptive. :+1: 


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @VicPeralta, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**